### DISCUSSION TITLE
Another tab for archived repositories

### DISCUSSION THREAD

**Discussion by tomazellifelipe** [community/community#41014] [Repositories]
### Select Topic Area

Product Feedback

### Body

I'd like to suggest that archived repositories show in another tab as I believe it would improve navigation, ux and ui of github website. I think that just a change in the web interface would not impact in other github resources that may depend on archived repositories be showed in searches like GitHub Code Search.

**Comment by jfullstackdev** [comment]
the shortest workaround is to save the filter through Chrome bookmark

here it is for your Archived tab

<https://github.com/tomazellifelipe?tab=repositories&q=&type=archived&language=&sort=>

anyways, you don't have public archives, that's why it's blank

but bookmark it and you're done

**Reply by tomazellifelipe** [reply]
The problem is more in a enterprise level where it tends to have way more archived repositories, but I guess, as you said, the shortest workaround is to use a better search filter.
